src/tests/routes/controllers/articles.test.js

```diff
@@ -83,17 +76,44 @@ describe('Articles', () => {
     it('should fail when all fields are not supplied', done => {
       chai
         .request(app)
-        .post('/api/v1/articles')
+        .post(endPoint)
         .set('x-access-token', jwtToken)
         .send(loginCredentials)
         .end((err, res) => {
           expect(res.status).to.equal(422);
+          expect(res.body.message).to.equal('validation error');
+          expect(res.body.errors).to.be.an('array');
+          done();
+        });
+    });
+    it('should fail when token is invalid or not supplied', done => {
+      chai
+        .request(app)
+        .post(endPoint)
+        .send(articleRequestObject)
+        .end((err, res) => {
+          expect(res.status).to.equal(401);
+          expect(res.body.status).to.equal('error');
           expect(res.body.message).to.equal(
-            'invalid/empty input. all fields must be specified.',
+            'You are not authorized to make this action',
           );
           done();
         });
     });
+
+    it('should create article successfully, with valid user input', done => {
+      chai
+        .request(app)
+        .post('/api/v1/articles')
+        .set('x-access-token', jwtToken)
+        .send(articleRequestObject)
+        .end((err, res) => {
+          expect(res.status).to.eql(201);
+          expect(res.body.status).to.eql('success');
+          expect(res.body.data).to.be.an('object');
```

Comment:
I think we agreed to test the content returned by the server when we send request body to it

Reply:
nice catch, thanks